tests/mmm/test_delayed_saturated_mmm.py

```diff
@@ -293,3 +293,17 @@ def test_data_setter(self, toy_X, toy_y):
             )
         except Exception as e:
             pytest.fail(f"_data_setter failed with error {e}")
+
+    def test_save_load(self, mmm_fitted):
+        model = mmm_fitted
+
+        model.save("test_save_load")
```

Comment:
shall we delete the file after the test ends? related to https://github.com/pymc-labs/pymc-marketing/issues/316

Reply:
I've added a line removing the temp file after the test is finished

Reply:
great! maybe we can remove the file of https://github.com/pymc-labs/pymc-marketing/issues/316 here :) ?

Reply:
I'm not sure what file we're talking about, I think the issue mentions something that was created and left on local of the person running the tests, not something I can do. But they shouldn't be left over after running tests anymore after that

Reply:
ah ok! thanks!